tests/src/org.freemarker/freemarker/2.3.31/src/test/resources/test.ftlh

```diff
@@ -0,0 +1,10 @@
+<html>
+<head>
+	<title>Welcome!</title>
+</head>
+<body>
+<h1>Welcome ${user}!</h1>
+<p>Our latest product:
+	<a href="${latestProduct.url}">${latestProduct.name}</a>!
+</body>
+</html>
```

Comment:
There should be a new line at the end of the file

Reply:
Done